packages/mux-player/src/index.ts

```diff
@@ -55,6 +55,7 @@ class MuxPlayerInternal {
     // }
 
     this._setUpErrors(el);
+    this._setUpLiveFirstPlay(el);
```

Comment:
Unless I'm missing something, this assumes playback-id has been set on initialization.

Reply:
Probably yes. As @luwes mentioned, this might be better served being moved into mux-video as well.
Though, there's also the question if we wanted to support the extended autoplay values.